src/auth/UserRequest.h

```diff
@@ -112,7 +112,7 @@ class UserRequest : public RefCountable
      \retval true   User has successfully been authenticated.
      \retval false  Timeouts on cached credentials have occurred or for any reason the credentials are not valid.
      */
-    virtual int authenticated() const = 0;
+    virtual bool authenticated() const;
```

Comment:
Please refactor PR title/description to focus on this API change rather than the bool/int polishing.

Reply:
done